### PR TITLE
Updating a_projectum to have a --notes field

### LINKED ISSUE #627: ENH: add notes field when add projectum

### PR DESCRIPTION
**Pull request by pliu2781**
closes #627

### CODE REVIEW

File: regolith/helpers/a_projectumhelper.py [hunk 1]
```diff
@@ -154,6 +160,10 @@ def db_updater(self):
             pdoc.update({
                 'collaborators': [],
             })
+        if rc.notes:
```
Comment (sbillinge) [inline regolith/helpers/a_projectumhelper.py:163]: if you set a default, is this if needed?

**Review by sbillinge — COMMENTED** [review]
looks good.  Please check the textarea widget to make sure it is the best.

**Comment by sbillinge** [comment]
also, we need to update the helper test to test this.

**Comment by connorjbracy** [comment]
@pliu2781's PR to close this issue, including subsequently request/added tests, has seemingly passed CI and been verified. Consider closing this issue or expanding what is left to be done.

**Comment by sbillinge** [comment]
> 
> 
> @pliu2781's PR to close this issue, including subsequently request/added tests, has seemingly passed CI and been verified. Consider closing this issue or expanding what is left to be done.

@pliu2781 @connorjbracy There is a review comment that is not addressed, so I am waiting for that.  There was also another one which is that I am not sure that TextArea widget is the right one for purpose and want that tested (or removed until we are convinced we need it)

File: regolith/helpers/a_projectumhelper.py (hunk 1 above)
Comment (pliu2781) [reply regolith/helpers/a_projectumhelper.py]: I removed the 'if'. Also, Textarea seems good when I checked it in helper_gui

**Review by sbillinge — COMMENTED** [review]
one last thing.  Please remove the Textarea widget for the notes.  I tested it on u_milestones and it breaks the code when you enter text on more than one line.  Please could you also remove it from u_milestones too, so it is just a regular text box (the default).

Then I can merge.  Thanks so much.

**Comment by sbillinge** [comment]
thanks @pliu2781

**Pull request merged**